### PR TITLE
test: set the right version for tests

### PR DESCRIPTION
**Pull request by shwetha-manvinkurke**
https://github.com/sendgrid/sendgrid-nodejs/pull/1329#discussion_r796108313

### CODE REVIEW

File: .github/workflows/test-and-deploy.yml [hunk 1]
```diff
@@ -33,6 +35,10 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
 
+      - name: Reset env vars
+        if: env.version == 'lts/*'
+        run: echo "version=lts" >> "$GITHUB_ENV"
+
       - run: npm install
```
Comment (childish-sambino) [inline .github/workflows/test-and-deploy.yml:42]: Don't need this step since it's run inside docker.

File: .github/workflows/test-and-deploy.yml [hunk 2]
```diff
@@ -33,6 +35,10 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
```
Comment (childish-sambino) [inline .github/workflows/test-and-deploy.yml:36]: Don't need this step since it's run inside docker.

File: .github/workflows/test-and-deploy.yml [hunk 3]
```diff
@@ -33,6 +35,10 @@ jobs:
...
 
+      - name: Reset env vars
+        if: env.version == 'lts/*'
```
Comment (childish-sambino) [inline .github/workflows/test-and-deploy.yml:39]: Can just change the matrix to use `lts` instead of `lts/*`.

Comment (shwetha-manvinkurke) [reply .github/workflows/test-and-deploy.yml:39]: That doesn't work

**Review by childish-sambino — APPROVED** [review]

**Pull request merged**